Core/ExchangeService.cs

```diff
@@ -5840,5 +5852,18 @@ internal string TargetServerVersion
         }
 
         #endregion
+        /// <summary>
+        /// Upload items to Exchange
+        /// </summary>
+        /// <param name="item">The item to Upload</param>
+        /// <returns></returns>
+        public UploadItemsResponse UploadItem(UploadItem item)

```

Comment:
Please remove the empty lines to follow the convention in the file.

Reply:
@richard-einfinity would be nice if you could fix that and sign the CLA again. We may get this into the official repo then. Thx!

Reply:
Return description updated and CLA resigned.